specs/posix/ssl_cert.table

```diff
@@ -0,0 +1,20 @@
+table_name("ssl_cert")
+description("SSL certificates information.")
+schema([
+    Column("domain", TEXT, "Issued organization domain name", required=True, hidden=True),
```

Comment:
Any reason for this to be hidden? I don't think it's really hurting anyone to display the domain name they've queried.